en/docs/get-started/cross-protocol-single-logout.md

```diff
@@ -0,0 +1,23 @@
+# Cross Protocol Single Logout
```

Comment:
This doc is not needed in the `get started` section. It should be there in `concepts`